package.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+    <name>cyclonedds</name>
+    <version>0.1.0</version>
+    <description>Eclipse Cyclone DDS is a very performant and robust open-source DDS implementation. Cyclone DDS is developed completely in the open as an Eclipse IoT project.</description>
+    <maintainer email="cyclonedds-dev@eclipse.org">Eclipse Foundation, Inc.</maintainer>
+    <license>Eclipse Public License 2.0</license>
+    <url type="website">https://projects.eclipse.org/projects/iot.cyclonedds</url>
+
+    <buildtool_depend>cmake</buildtool_depend>
```

Comment:
Nitpick: we try to keep any kind of enumeration like these `buildtool_depend` in alphabetical order since it makes any kind of insert/remove deterministic and helps users visually browse the list.